### PR TITLE
Condor Singularity with Conda for TPV

### PR DESCRIPTION
**Pull request by mira-miracoli**
A destination for tools that need singularity and conda as resolver, as mentioned in [#356](https://github.com/usegalaxy-eu/issues/issues/356).
Also added the tool [enasearch_data](https://toolshed.g2.bx.psu.edu/repository?repository_id=cd6d54fc1874dc60) which is already in IUC but needs to be installed in our Galaxy.

### CODE REVIEW

File: files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/destinations.yml [hunk 1]
```diff
@@ -63,3 +63,31 @@ destinations:
...
+      request_cpus: "{PARALLELISATION}"
+      request_memory: "{MEMORY}"
+      requirements: 'GalaxyGroup == "compute"'
+      tmp_dir: "True"
+      metadata_strategy: "extended"
+      require_container: true
+      singularity_enabled: true
+      # can and should be made stricter at some point
+      singularity_volumes: "$_CONDOR_SCRATCH_DIR:rw,$job_directory:rw,$tool_directory:ro,$job_directory/outputs:rw,$working_directory:rw,/data/db/:ro,/data/dnb01/galaxy_db/:ro,/data/dnb02/galaxy_db/:ro,/data/dnb03/galaxy_db/:ro,/data/dnb05/galaxy_db/:ro,/data/dnb06/galaxy_db/:rw,/data/dnb07/galaxy_db/:rw,/data/dp01/galaxy_db/:rw,/data/0/galaxy_db/:ro,/data/1/galaxy_db/:ro,/data/2/galaxy_db/:ro,/data/3/galaxy_db/:ro,/data/4/galaxy_db/:ro,/data/5/galaxy_import/galaxy_user_data/:ro,/data/6/galaxy_db/:ro,/data/7/galaxy_db/:ro,/usr/local/tools/:ro"
+      container_override:
```
Comment (bgruening) [inline files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/destinations.yml:89]: I'm not sure TPV will take this ... maybe but there is no other way than testing it :)

File: files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/destinations.yml [hunk 2]
```diff
@@ -63,3 +63,31 @@ destinations:
...
+        - type: singularity
+          shell: "/bin/bash"
+          resolve_dependencies: true
+          identifier: "/cvmfs/singularity.galaxyproject.org/all/centos:8.3.2011"
```
Comment (bgruening) [inline files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/destinations.yml:93]: Once we get the destination up and running this needs to be a Python2 by default container and maybe we should reflect this in the destination name as well?

File: files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/destinations.yml (hunk 1 above)
Comment (mira-miracoli) [reply files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/destinations.yml:89]: According to this page, all entities should share these [https://total-perspective-vortex.readthedocs.io/en/latest/topics/concepts.html](https://total-perspective-vortex.readthedocs.io/en/latest/topics/concepts.html)

File: files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/destinations.yml (hunk 2 above)
Comment (mira-miracoli) [reply files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/destinations.yml:93]: I can add this, yes

**Pull request merged**